shared/actions/app.js

```diff
@@ -1,40 +1,14 @@
 // @flow
-import logger from '../logger'
-import * as RPCTypes from '../constants/types/rpc-gen'
 import * as Saga from '../util/saga'
 import * as AppGen from './app-gen'
 import {showMainWindow} from './platform-specific'
 
-function _onMobileAppStateChanged(action: AppGen.MobileAppStatePayload) {
-  const nextAppState = action.payload.nextAppState
-
-  const appFocused = {
-    active: true,
-    background: false,
-    inactive: false,
-  }[nextAppState]
-
-  const state =
-    {
-      active: RPCTypes.appStateAppState.foreground,
-      background: RPCTypes.appStateAppState.background,
-      inactive: RPCTypes.appStateAppState.inactive,
-    }[nextAppState] || RPCTypes.appStateAppState.foreground
-  logger.info(`setting app state on service to: ${state}`)
-
-  return Saga.sequentially([
-    Saga.put(AppGen.createChangedFocus({appFocused})),
```

Comment:
we need to make sure removing this doesn't break anything right

Reply:
oof yeah, it's really just the RPC that should be removed here. thanks